<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_AllDaq</class>
 <widget class="QDialog" name="Dial_AllDaq">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAQ All Module </string>
  </property>
  <widget class="QCheckBox" name="checkBox_Amc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Amc</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_0">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Thor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Thor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Start">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>440</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>498</y>
     <width>771</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>Select</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>141</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Asm</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 6</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 7</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 8</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 9</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 10</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 11</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_All">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Clear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_seqON_0">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sequence &amp;0</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_seqON_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sequence &amp;1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_seqON_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sequence &amp;2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_seqON_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sequence &amp;3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_seqON_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sequence &amp;4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_seqON_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sequence &amp;5</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_seqON_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Si&amp;multaneous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Stop">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>191</width>
     <height>211</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="radioButton_seqOFF_0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Se&amp;quence 0</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_seqOFF_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequence 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_seqOFF_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequence 2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_seqOFF_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequence 3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_seqOFF_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequence 4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_seqOFF_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequence 5</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_seqOFF_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Simul&amp;taneous</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>161</width>
     <height>231</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-20</y>
      <width>221</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Start Sequence</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>221</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Start Sequence</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>221</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Stop Sequence</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_stat">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>280</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Daq Status</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="delayTrigthor">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Label_delay_trig">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay Trigger</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ModeDaqThor">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mode Thor And Left-Right</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Coinc 2 Mezza</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Coinc 1 to 3 board</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor Piedestaux</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor Or</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Label_freqPiedthor">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Freq Pied Thor</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_freqPiedthorCalcul">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(2n+2)*20µs</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="FreqPiedThor">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QLabel" name="Label_DelayCalcul">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>380</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(2n+2)*20µs</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="AsmModeThor">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>300</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm mode Thor</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="EnableRoi">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable Roi</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NbSamples">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
  </widget>
  <widget class="QLabel" name="Label_delay_trig_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NbSamples</string>
   </property>
  </widget>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>widget_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>checkBox_Amc</zorder>
  <zorder>checkBox_Asm_0</zorder>
  <zorder>checkBox_Thor</zorder>
  <zorder>pushButton_Start</zorder>
  <zorder>pushButton_Close</zorder>
  <zorder>textEdit_Msg</zorder>
  <zorder>checkBox_Asm_1</zorder>
  <zorder>checkBox_Asm_2</zorder>
  <zorder>checkBox_Asm_3</zorder>
  <zorder>checkBox_Asm_4</zorder>
  <zorder>checkBox_Asm_5</zorder>
  <zorder>checkBox_Asm_6</zorder>
  <zorder>checkBox_Asm_7</zorder>
  <zorder>checkBox_Asm_8</zorder>
  <zorder>checkBox_Asm_9</zorder>
  <zorder>checkBox_Asm_10</zorder>
  <zorder>checkBox_Asm_11</zorder>
  <zorder>checkBox_All</zorder>
  <zorder>checkBox_Clear</zorder>
  <zorder>radioButton_seqON_0</zorder>
  <zorder>radioButton_seqON_1</zorder>
  <zorder>radioButton_seqON_2</zorder>
  <zorder>radioButton_seqON_3</zorder>
  <zorder>radioButton_seqON_4</zorder>
  <zorder>radioButton_seqON_5</zorder>
  <zorder>radioButton_seqON_6</zorder>
  <zorder>pushButton_Stop</zorder>
  <zorder>widget</zorder>
  <zorder>lineEdit_stat</zorder>
  <zorder>label</zorder>
  <zorder>delayTrigthor</zorder>
  <zorder>Label_delay_trig</zorder>
  <zorder>ModeDaqThor</zorder>
  <zorder>Label_freqPiedthor</zorder>
  <zorder>Label_freqPiedthorCalcul</zorder>
  <zorder>FreqPiedThor</zorder>
  <zorder>Label_DelayCalcul</zorder>
  <zorder>AsmModeThor</zorder>
  <zorder>EnableRoi</zorder>
  <zorder>NbSamples</zorder>
  <zorder>Label_delay_trig_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
